suite-common/wallet-utils/src/__fixtures__/transactionUtils.ts

```diff
@@ -1,11 +1,22 @@
 import { testMocks } from '@suite-common/test-utils';
 import { WalletAccountTransaction } from '@suite-common/wallet-types';
 import { AccountTransaction } from '@trezor/connect';
+import { TransferType } from '@trezor/blockchain-link-types';
 
 import { TXS } from './transactions';
 
 const { getWalletAccount } = testMocks;
 
+export const token = {
```

Comment:
In general I think for any object (regardless if its a fixture) we shall have declarative typing

here its probably: `export const token: TokenTransfer = {`

Benefits of this approach are: 
- better IDE support and auto-complete
- errors in types are reported by Typescript at its source and not somewhere else (when this value is passed).

Reply:
Agree.

Just sometimes for fixtures it may not make much sense since you need for testing just for values from the object (and it can be really big and complicated) but I will add it here since it was just screaming it's missing contract property.